File: WebSpeak v.1/VitaliiMahlona/DAL/Models/DTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models
{
    public class DTO
    {
        public int Id { get; set; }
        public string WordNativeLang { get; set; }
        public string WordLearnLang { get; set; }
        public string Picture { get; set; }
        public string Sound { get; set; }
        public string PronounceLearn { get; set; }
        public string PronounceNative { get; set; }
        public int? SubCategoryId { get; set; }
        public int? CategoryId { get; set; }
        public string EnableNativeLang { get; set; }
        public string EnableSound { get; set; }
        public string EnablePronounceNativeLang { get; set; }
        public string EnablePronounceLearnLang { get; set; }
        public int TestId { get; set; }
        public int Total { get; set; }
        public string UserId { get; set; }
    }
}

File: WebSpeak v.2/VitaliiMahlona/LearningLanguagesAngular/ClientApp/app/models/DTO.ts
import { Languages } from "./Languages"

export class DTO {
    public id: number;
    public idLangNative: number;
    public idLangLearn: number;
    public testId: number;
    public total: number;
    public languagesList: Languages[];
    public wordNativeLang: string;
    public wordLearnLang: string;
    public userId: string;
    public picture: string;
    public enableNativeLang: string;
    public enableSound: string;
    public enablePronounceNativeLang: string;
    public enablePronounceLearnLang: string;
    public sound: string;
    public pronounceLearn: string;
    public pronounceNative: string;
    public categoryId?: number;
    public subCategoryId?: number;
}